core/storage/rocksdb/rocksdb.hpp

```diff
@@ -99,7 +101,7 @@ namespace kagome::storage {
     outcome::result<std::shared_ptr<RocksDb>> use() const;
 
     std::weak_ptr<RocksDb> storage_;
-    RocksDb::ColumnFamilyHandle column_;
+    RocksDb::ColumnFamilyHandlePtr column_;
```

Comment:
Must be valid after `RocksDb::dropColumn` (element ref or vector index).
```suggestion
    const RocksDb::ColumnFamilyHandlePtr &column_;
```

Reply:
Yes, it will. It used by pointer-to-pointer, and will be updated in-place.